### PR TITLE
#784 app display name setting

### PR DESCRIPTION
**Pull request by Dzejkop**

### CODE REVIEW

**Comment by Dzejkop** [comment]
I need to integrate this into a scene, so don't merge yet.

File: Assets/PatchKit Patcher/Scripts/GameTitle.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using UnityEngine.UI;
+using UniRx;
+using PatchKit.Unity.Patcher.Debug;
+
+namespace PatchKit.Unity
+{
+    public class GameTitle : MonoBehaviour
+    {
+        [SerializeField]
+        private Text targetText;
```
Comment (genail) [inline Assets/PatchKit Patcher/Scripts/GameTitle.cs]: Private fields should look like this: `_targetText`.

File: Assets/PatchKit Patcher/Scripts/GameTitle.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        void Start()
+        {
+            var patcher = Patcher.Patcher.Instance;
```
Comment (genail) [inline Assets/PatchKit Patcher/Scripts/GameTitle.cs]: Please do not use vars if type cannot be easily guessed in the same line.

Comment (genail) [inline Assets/PatchKit Patcher/Scripts/GameTitle.cs]: Assert to check if targetText is set would be nice.

**Comment by witcher112** [comment]
I've made minor refactor to show you more reactive solution @Dzejkop , take a look at it :)

**Comment by witcher112** [comment]
@genail if everything's okay, please merge :)

**Comment by witcher112** [comment]
@genail bump

**Pull request merged**